packages/fxa-payments-server/src/components/PayPalButton/index.tsx

```diff
@@ -210,27 +211,33 @@ export const PaypalButton = ({
     tagline: 'false',
   };
 
+  const disabledStyles =
+    ' payment-button-disabled after:bg-white after:opacity-40 after:z-[100]';
+
   return (
-    <div className="w-60 h-24 m-auto relative z-0">
-      <div
-        className={
-          disabled
-            ? "relative after:absolute after:bg-white after:content-[''] after:opacity-60 after:top-0 after:left-0 after:w-full after:h-full after:z-[1000]"
-            : undefined
-        }
-        data-testid="paypal-button-container"
-      >
-        {ButtonBase && (
-          <ButtonBase
-            style={styleOptions}
-            data-testid="paypal-button"
-            createOrder={createOrder}
-            onInit={onInit}
-            onApprove={onApprove}
-            onError={onError}
-          />
-        )}
-      </div>
+    <div data-testid="pay-with-other">
+      <Localized id="pay-with-heading-paypal">
```

Comment:
Let's add an `en.ftl` in `components/PayPalButton` and move the ftl id in there for consistency.